lib/components/Upload.js

```diff
@@ -119,6 +137,32 @@ export default class Upload extends Component {
     }
   }
 
+  onMedtronicSerialNumberInputChange() {
+    const { refs } = this;
+    let serial_number = refs.medtronic_serial_number;
```

Comment:
No specific reason, that's just how the CareLink inputs are handled, so I used the same for consistency.